pravega-cli/src/main/java/io/pravega/tools/pravegacli/commands/admin/PasswordFileCreatorCommand.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.pravega.tools.pravegacli.commands.admin;
+
+import com.google.common.base.Strings;
+import io.pravega.tools.pravegacli.commands.CommandArgs;
+import lombok.Cleanup;
+import io.pravega.tools.pravegacli.commands.Command;
+import io.pravega.controller.server.rpc.auth.StrongPasswordProcessor;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.lang.ArrayIndexOutOfBoundsException;
+
+public class PasswordFileCreatorCommand extends Command {
+    static final String COMPONENT = "admin";
+    public PasswordFileCreatorCommand(CommandArgs args){super(args);}
```

Comment:
Do not interleave attributes and methods/constructors. I recommend you to have a look to this doc: https://github.com/pravega/pravega/wiki/Contributing#codestyle

Reply:
Will make `COMPONENT`  local to CommandDescriptor.